src/client/Microsoft.Identity.Client/ApiConfig/AcquireTokenForClientParameterBuilder.cs

```diff
@@ -74,6 +74,16 @@ public AcquireTokenForClientParameterBuilder WithSendX5C(bool withSendX5C)
             return this;
         }
 
+        /// <summary>
+        /// Specifies that the certificate provided will be used for PoP tokens with MTLS (Mutual TLS) authentication.
+        /// </summary>
+        /// <returns>The current instance of <see cref="AcquireTokenForClientParameterBuilder"/> to enable method chaining.</returns>
+        public AcquireTokenForClientParameterBuilder WithMtlsPop()
```

Comment:
Let's soft deprecate and hide the existing POP method and create a new `WithSignedHttpRequestProofOfPosession` or `WithShrProofOfPosession` ? 

CC @localden

Reply:
Same PR or should that be a different task? @localden

Reply:
We can do this in the same PR - group the changes. I like the explicit API naming (`WithSignedHttpRequestProofOfPosession`)